Content.Client/Atmos/UI/GasVolumePumpWindow.xaml

```diff
@@ -0,0 +1,23 @@
+<SS14Window xmlns="https://spacestation14.io"
```

Comment:
there's a lot of duplication between the volume pump and pressure pump but they're both so small that I don't think I really care